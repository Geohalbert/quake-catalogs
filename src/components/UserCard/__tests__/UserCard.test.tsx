import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

import QuakeCard from '../index';

describe('<QuakeCard />', () => {
  it('renders', () => {
    const mockData = {
      name: 'Welly',
      phone: '007',
      email: 'test@gmail.com',
      website: 'www.test.com'
    };
    const tree = renderer
      .create(
        <MemoryRouter>
          <QuakeCard info={mockData} />
        </MemoryRouter>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
