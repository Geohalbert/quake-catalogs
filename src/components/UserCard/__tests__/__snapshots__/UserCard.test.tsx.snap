// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<QuakeCard /> renders 1`] = `
<div
  className="QuakeCard"
>
  <h4>
    Quake Card
  </h4>
  <ul>
    <li>
      Name: 
      Welly
    </li>
    <li>
      Phone: 
      007
    </li>
    <li>
      Email: 
      test@gmail.com
    </li>
    <li>
      Website: 
      www.test.com
    </li>
  </ul>
</div>
`;
